Analysis/Tasks/PWGCF/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@
 # or submit itself to any jurisdiction.
 
 add_subdirectory(FemtoDream)
+add_subdirectory(GenericFramework)
```

Comment:
Shouldn't this directory be named to something more specific, like "FlowGenericFramework" ?